tests/Fixtures/DummyTranslation.php

```diff
@@ -5,38 +5,28 @@
 namespace Locastic\ApiPlatformTranslationBundle\Tests\Fixtures;
 
 use Locastic\ApiPlatformTranslationBundle\Model\AbstractTranslation;
+use Locastic\ApiPlatformTranslationBundle\Model\TranslatableInterface;
 
 /**
- * Class DummyTranslation
- *
  * @package Locastic\ApiPlatformTranslationBundle\Tests\Fixtures
  */
 class DummyTranslation extends AbstractTranslation
 {
-    /**
-     * @var null|string
-     */
-    private $translation;
+    public function __construct(
```

Comment:
@SpartakusMd this will not be needed if we remove the extra added constructor in the AbstractTranslation

Reply:
It was added by the original author of the PR. Removing this constructor as it's not needed indeed.